uwblueprint/starter-code-v2
# chore(deps): update Mongoose to v6.2.2

## Notion ticket link

[Update Mongoose to v6.x](https://app.zenhub.com/workspaces/internal-tools-60bd08056aec7b00103160ab/issues/uwblueprint/starter-code-v2/147)



## Implementation description

I've gone through all the documentation for the upgrade and I think I've got it all! See comment in ticket on zenhub

1. Only a few changes needed to be made - including some outside of typescript/models
2. string[] was replaced with [string] in the backend because after the mongoose update the backend threw errors



## Steps to test
1. Make sure it compiles and builds without errors. Test to make sure you can access the frontend
2. Ensure that the mongo connection was created successfully, many changes in v6 occurred with the connection code




## What should reviewers focus on?
* 
Please take a look at the yarn.lock. Do I even need these changes? Why so many line changes? Try running yarn install yourself to see if you can lower the line count. 

## Checklist
